code/modules/client/preference/preferences.dm

```diff
@@ -252,7 +249,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 				S = GLOB.all_species[species]
 				random_character()
 
-			dat += "<div class='statusDisplay' style='max-width: 128px; position: absolute; left: 150px; top: 150px'><img src=previewicon.png class='charPreview'><img src=previewicon2.png class='charPreview'></div>"
+			dat += "<div class='statusDisplay'style='position: absolute; left: 170px; top: 110px;'><img src=previewicon.png width=[preview_icon.Width()] height=[preview_icon.Height()]></div>"
```

Comment:
Not gonna make you try to fix this html so that it doesn't look awful on the character panel, but I'm going to leave a note here that it should be done.